### PR TITLE
[10.0] account_financial_report_qweb: supplier ref in general ledger

### PR DESCRIPTION
**Pull request by p-tombez**
Display supplier reference in column label in general ledger report

### CODE REVIEW

File: account_financial_report_qweb/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     'name': 'QWeb Financial Reports',
-    'version': '10.0.3.1.0',
+    'version': '10.0.3.0.1',
```
Comment (guewen) [inline account_financial_report_qweb/__manifest__.py]: Hmm?

```suggestion
    'version': '10.0.3.2.0',
```

**Review by guewen — CHANGES_REQUESTED** [review]
LGTM but there is a problem with the version.

**Review by asaunier — APPROVED** [review]
Try perhaps to rebase the branch in order to fix Travis errors.

**Review by asaunier — APPROVED** [review]

**Review by vrenaville — APPROVED** [review]

**Review by guewen — APPROVED** [review]

File: account_financial_report_qweb/__manifest__.py [hunk 2]
```diff
@@ -16,7 +16,8 @@
     "website": "https://odoo-community.org/",
     'depends': [
         'account',
-        'account_group',    # account-financial-tools
+        'account_group',                        # account-financial-tools
+        'account_invoice_supplier_ref_unique',  # account-invoicing
```
Comment (pedrobaeza) [inline account_financial_report_qweb/__manifest__.py:20]: You can't change dependencies in a stable version, and more adding this dependency that is not mandatory. You must do a bridge module if you want that data.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Pull request closed without merging**